R/chk_description.R

```diff
@@ -27,7 +27,7 @@ CHECKS$no_description_date <- make_check(
   tags = "DESCRIPTION",
   preps = "description",
 
-  gp = 'node use "Date" in DESCRIPTION. It is not required and it
+  gp = 'do not use "Date" in DESCRIPTION. It is not required and it
```

Comment:
This is obviously a typo, but the output always starts with 

> It is good practice to

so in this case it could continue as `'omit "Date" in DESCRIPTION...`